### PR TITLE
Prepare v5 release

### PR DESCRIPTION
**Pull request by daun**
**Description**

- Bump version
- Add changelog item
- Update readme

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+## [5.0.0] - 2024-04-15
+
+This is a complete overhaul of the plugin, based on extensive feedback and testing in screen
+readers. Some breaking changes have been made to improve the defaults. If you haven't customized the
+plugin options, it's safe to upgrade, otherwise consult the readme.
+
+- Rethink announcements: use assertive live region and announce after a delay (@ScoobyDid)
+- Focus `body` on navigation instead of `main`
+- Only use `h1` for announcements, ignore `h2`
+- Disable animations if users prefer reduced motion
```
Comment (hirasso) [inline CHANGELOG.md:12]: Disable animations _by default_ if ...?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 
+## [5.0.0] - 2024-04-15
+
+This is a complete overhaul of the plugin, based on extensive feedback and testing in screen
+readers. Some breaking changes have been made to improve the defaults. If you haven't customized the
+plugin options, it's safe to upgrade, otherwise consult the readme.
+
+- Rethink announcements: use assertive live region and announce after a delay (@ScoobyDid)
+- Focus `body` on navigation instead of `main`
+- Only use `h1` for announcements, ignore `h2`
```
Comment (hirasso) [inline CHANGELOG.md:11]: By default, only use the `h1` for announcements, ...?

**Review by hirasso — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (daun) [reply CHANGELOG.md]: 👌

File: CHANGELOG.md (hunk 2 above)
Comment (daun) [reply CHANGELOG.md]: 👌👌

**Comment by daun** [comment]
@hirasso I'm currently doing some manual testing and it looks like VoiceOver is now refusing to read the new page title. Will investigate further and do a proper round of full manual testing before going anywhere near a release 🤠

**Comment by hirasso** [comment]
How good that you caught this! Sure, take your time.

**Comment by daun** [comment]
@hirasso It just didn't announce them. I suspect because it's somehow busy and just dropping the announcement, something we didn't have before. Might have to do with the move to `visit:end` in combination with some other detail. The text is definitely being added to the live region, so that can't be it. Impossible to catch this stuff in automatic tests :(

**Pull request merged**